src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+  let total = 0;
+
+  if (days < 3) {
+    total = days * dailyRent;
+  } else if (days >= 3 && days < 7) {
+    total = days * dailyRent - minDiscount;
+  } else {
```

Comment:
The third rule of checklist
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

Reply:
thank you, everything's corrected